css/index.css

```diff
@@ -0,0 +1,61 @@
+
+@media screen and (max-width: 736px) {
+
+}
+
+body {
+    background-color: rgb(185, 219, 250);
+    color: #160181;
+    font-family: "Montserrat", sans-serif;
+}
+section {
+   margin: 2rem;
+   padding: 2rem; 
+}
+h2 {
+    font-size: 1.5rem;
+    color: #0c033d4d;
+    font-weight: bold;
+}
```

Comment:
All this looks great so far! Knocking out the requirements methodically. Good stuff.